leasing/report/invoice/invoicing_review.py

```diff
@@ -270,21 +287,22 @@ def get_incorrect_rent_shares_data(self, cursor):
                            INNER JOIN leasing_tenantcontact tc
                            ON t.id = tc.tenant_id
                               AND tc.type = 'tenant'
-                              AND tc.start_date <= %(today)s
-                              AND (tc.end_date IS NULL OR tc.end_date >= %(today)s)
+                              AND tc.start_date <= '{today}'
+                              AND (tc.end_date IS NULL OR tc.end_date >= '{today}')
                               AND tc.deleted IS NULL
                            INNER JOIN leasing_tenantrentshare trs
                            ON t.id = trs.tenant_id
                               AND trs.deleted IS NULL
                      WHERE t.deleted IS NULL
                    ) tt ON tt.lease_id = l.id
-            WHERE (l.end_date IS NULL OR l.end_date >= %(today)s)
+            WHERE (l.end_date IS NULL OR l.end_date >= '{today}')
               AND l.deleted IS NULL
+              {service_unit_filter}
             GROUP BY l.id,
                      li.id;
         """
 
-        cursor.execute(query, {"today": today})
+        cursor.execute(query.format(today=today, service_unit_filter=service_unit_filter))
```

Comment:
Is the formatted the same way now that the way it is handled changed? I know it should be in the same format, but have you checked/tested it?